File: TransCoder_tokenized_test_set_functions/cpp/CHECK_WHETHER_POINT_EXISTS_CIRCLE_SECTOR_NOT.cpp
void checkPoint ( int radius, int x, int y, float percent, float startAngle ) {
  float endAngle = 360 / percent + startAngle;
  float polarradius = sqrt ( x * x + y * y );
  float Angle = atan ( y / x );
  if ( Angle >= startAngle && Angle <= endAngle && polarradius < radius ) printf ( "Point (%d, %d) exist in the circle sector\n", x, y );
  else printf ( "Point (%d, %d) does not exist in the circle sector\n", x, y );
}

File: TransCoder_tokenized_test_set_functions/python/CHECK_WHETHER_POINT_EXISTS_CIRCLE_SECTOR_NOT.py
def checkPoint ( radius, x, y, percent, startAngle ) :
    endAngle = 360 / percent + startAngle
    polarradius = math . sqrt ( x * x + y * y )
    Angle = math . atan ( y / x )
    if ( Angle >= startAngle and Angle <= endAngle and polarradius < radius ) :
        print ( "Point (", x, ",", y, ") " "exist in the circle sector" )
    else :
        print ( "Point (", x, ",", y, ") " "does not exist in the circle sector" )
    
